insta_reels_publishing_api_sample/.env.template

```diff
@@ -4,3 +4,11 @@ API_SECRET=YOUR_APP_SECRET
 REDIRECT_URI=https://localhost:8000/callback
 SESSION_SECRET=RANDOM_SESSION_SECRET_STRING
 PORT=8000
+
+# Optional config setting to change the Graph API version.
+# Leave blank or commented to default to the app's default version
+# GRAPH_API_VERSION_PATH=v18
+
+# Optional config setting to chage the Graph API host
+# Leave blank or commented to default to the default Graph API host name
+# DEFAULT_GRAPH_API_ORIGIN=https://graph.facebook.com
```

Comment:
Shouldn't this be simply `GRAPH_API_ORIGIN`?

Reply:
Nice catch!